shell/platform/darwin/common/framework/Headers/FlutterBaseDartProject.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERBASEDARTPROJECT_H_
+#define FLUTTER_FLUTTERBASEDARTPROJECT_H_
+
+#import <Foundation/Foundation.h>
+
+#import "FlutterMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * A set of Flutter and Dart assets used by a `FlutterEngine` to initialize execution.
+ *
+ * TODO(stuartmorgan): Align API with iOS DartProject, and combine.
```

Comment:
Should this be removed?

Reply:
No. We still eventually need this.

Reply:
Perhaps this comment should be on each embedder's type and not the shared common one? I don't feel strongly though, feel free to keep as is